package.json

```diff
@@ -20,13 +31,15 @@
         "body-parser": "^1.19.0",
         "bootstrap": "^4.4.1",
         "byline": "^5.0.0",
+        "chalk": "^2.4.2",
         "chart.js": "^2.9.3",
         "cheerio": "^0.22.0",
         "clipboard": "^2.0.6",
         "cookie-parser": "^1.4.5",
         "crypto-js": "^4.0.0",
         "csv": "^5.3.2",
         "csvtojson": "^2.0.10",
+        "date-fns": "^2.0.0",
```

Comment:
Is this doing something different/better than `moment.js`? I'm just checking in a losing attempt to keep our library dependencies under control 😄

Reply:
The "different/better" is "functional interface that's more ergonomic" 😂 I forget exactly why I installed this instead of using moment, happy to back this out.